code/modules/admin/verbs/event_triggers.dm

```diff
@@ -0,0 +1,121 @@
+							return
+					M.forceMove(get_turf(ET))
+				if(decision == "Delete")
+					var/confirm = tgui_alert(src, "ARE YOU SURE? THERE IS NO GOING BACK FROM DELETING [ET.name]", "CONFIRM", list("Go Back", "Delete it!"), autofocus = FALSE)
+					if(confirm == "Go Back")
+						return
+					ET.delete_me = TRUE
+					qdel(ET)
+		if("Manage Other's Triggers")
+			var/other_ckey = sanitize(tgui_input_text(src, "input trigger owner's ckey", "CKEY", ""))
+			var/others_list = event_triggers[other_ckey]
+			if(!LAZYLEN(others_list))
+				to_chat(src, SPAN_NOTICE("[other_ckey] doesn't have any landmarks to manage!"))
+				return
+			others_list |= list("Cancel", "Delete All")
+			var/choice = tgui_input_list(src, "Select a landmark to choose between teleporting to it or deleting it, select delete all to clear them.", \
+			"Manage Personal Triggers", others_list)
+			if(choice == "Cancel")
+				return
+			if(choice == "Delete All")
+				if(other_ckey && GLOB.directory[other_ckey])
+					var/client/C = GLOB.directory[other_ckey]
+					var/mob/M = C.statobj
+					if(M.client && M.client.inactivity < 30 MINUTES)
+						to_chat(src, SPAN_WARNING("[M] has only been inactive for [M.client.inactivity / (1 MINUTE)] minutes.\n \
+						If you want to delete their event triggers, ask them in asay or discord to do it themselves or wait 30 minutes"))
+						log_and_message_admins("[src.ckey] tried to delete all of [other_ckey]'s event triggers, but [other_ckey] is active.")
+						return
+				var/confirm = tgui_alert(src, "ARE YOU SURE? THERE IS NO GOING BACK", "CONFIRM", list("Go Back", "Delete all my event triggers"), autofocus = FALSE)
+				if(confirm == "Go Back")
+					return
+				for(var/obj/effect/landmark/event_trigger/ET in others_list)
+					ET.delete_me = TRUE
+					qdel(ET)
+				log_and_message_admins("[src.ckey] deleted all of [other_ckey]'s event triggers. [other_ckey] was either inactive or disconnected at this time.")
+			else if(istype(choice, /obj/effect/landmark/event_trigger))
+				var/obj/effect/landmark/event_trigger/ET = choice
+				var/decision = tgui_alert(src, "Teleport to Landmark or Delete it?", "Manage [ET]", list("Teleport", "Delete"), autofocus = FALSE)
+				if(decision == "Teleport")
+					var/mob/M = usr
+					if(isobserver(M))
+						var/confirm_teleport = tgui_alert(src, "You're not a ghost! Admin-ghost?", "You're not a ghost", \
+							list("Cancel", "Teleport me with my character"))
+						if(confirm_teleport == "Cancel")
+							return
+					M.forceMove(get_turf(ET))
+				if(decision == "Delete")
+					if(other_ckey && GLOB.directory[other_ckey])
+						var/client/C = GLOB.directory[other_ckey]
+						var/mob/M = C.statobj
+						if(M.client && M.client.inactivity < 30 MINUTES)
+							to_chat(src, SPAN_WARNING("[M] has only been inactive for [M.client.inactivity / (1 MINUTE)] minutes.\n \
+							If you want to delete their event triggers, ask them in asay or discord to do it themselves or wait 30 minutes"))
```

Comment:
This is up for staff debate tbh, but I do think having some authority override on ability to delete those should be present.

Reply:
Wouldn't kicking a non-compliant staff creating abusive landmarks handle this?

Reply:
On general level yes, but again, this is something for staff to discuss as a whole. I just think that, code-side, these things shouldnt be hardcoded like that, with 0 regard for hiearchy or potential abuse.